director/authentication/json_web_token.py

```diff
@@ -0,0 +1,31 @@
+from flask import jsonify, request
+import jwt 
+import datetime
+from functools import wraps
+secret = 'pass'
+
+"""
+THIS function requires the use of a json web token in order to 
+be accepted 
+"""
+def token_required(f):
+    @wraps(f)
+    def decorated(*args, **kwargs):
+        token = None
+        if 'x-access-token' in  request.headers:
+            token = request.headers['x-access-token']
+        if not token:
+            jsonify({'message': 'Token is missing'}), 401
+        try:
+            data = jwt.decode(token, secret)
+        except:
+            return jsonify({'message':'Token is false'}), 401
+        return f(*args,**kwargs)
+    return decorated
+
+
+def generate_Jwt(username):
```

Comment:
Please be consistent in casing! 😄 This should be `generate_jwt`.
On top of that, this method should be somewhere in the `api` module.

Reply:
done